connect/runtime/src/test/java/org/apache/kafka/connect/integration/ExampleConnectIntegrationTest.java

```diff
@@ -54,9 +55,8 @@
 
     private static final int NUM_RECORDS_PRODUCED = 2000;
     private static final int NUM_TOPIC_PARTITIONS = 3;
-    private static final int CONSUME_MAX_DURATION_MS = 5000;
+    private static final int RECORD_MAX_DURATION_MS = 5000;
```

Comment:
would `RECORD_TRANSFER_MAX_DURATION` be a better name here? `RECORD_MAX_DURATION_MS` seemed to indicate that we are recording the max duration.